workspace_tools/export/exporters.py

```diff
@@ -12,12 +12,99 @@
 from workspace_tools.toolchains import TOOLCHAIN_CLASSES
 from workspace_tools.targets import TARGET_MAP
 
+from project_generator.generate import Generator
+from project_generator.project import Project
+from project_generator.settings import ProjectSettings
+
 class OldLibrariesException(Exception): pass
 
-class Exporter():
+class Exporter(object):
     TEMPLATE_DIR = dirname(__file__)
     DOT_IN_RELATIVE_PATH = False
 
+    # mapping mbed to progen targets
+    PROGEN_TARGETS = {
```

Comment:
Have you thought about moving this definition into the target's definition in targets.py?

It could either be a separate attribute, like `self.exporters_supported = { 'progen': 'mbed-lpc1768' }` or `self.exporters_supported = { 'progen': True }` (to use the default name) for the LPC1768. Not too sure how it should look, but I'm worried about maintaining a separate list of target mappings that people can miss when they're adding new platforms.